data/mods/blazemod/blaze_blob_parts.json

```diff
@@ -1753,174 +1009,51 @@
     "damage_modifier": 10,
     "folded_volume": 1,
     "breaks_into": [ { "item": "glooplegrow" } ],
-    "requirements": {
-      "repair": {
-        "time": 30000,
-        "using": [ [ "blobrep", 1 ] ]
-      }
-    },
+    "description": "A living blog, formed into a half-height vehicle wall.  Keeps zombies outside the vehicle but allows people to see over it.",
```

Comment:
Night of the `living blog` sounds horrible.
Should be `blob`

Reply:
it does sound horrible.
fixed.